### PR TITLE
Pw/add cmd command

### PR DESCRIPTION
**Pull request by philipwhittingham**

### CODE REVIEW

File: bin/dev [hunk 1]
```diff
@@ -72,6 +73,16 @@ HEREDOC
...
   end
+
+  def cmd(args)
+    args = ["/bin/bash", "-l"] if args.empty?
+    container_id = `docker container ls -q -f name=ermanagement_web_`.strip
+    if container_id == ""
+      puts "Container is not running"
+    else
+      system("docker exec -i -t #{container_id} #{args.join(" ")}")
+    end
```
Comment (wgrrrr) [inline bin/dev]: We can convert this to work more like `dev run worker bash` if no args are present, or `dev run worker {something}` if args are present.

**Review by wgrrrr — CHANGES_REQUESTED** [review]

File: bin/dev [hunk 2]
```diff
@@ -28,12 +28,14 @@ Commands:
...
   proxy              Manage proxy services (nginx and dnsmasq)
+  webinject          Open/Run a command line in the web service
+  cmd                Open/Run a command in a new app container
 
 HEREDOC
 
   PROXY_UNMODIFIED_TO_COMPOSE=%w{build kill logs ps pull restart rm start stop}
   PROXY_MODIFIED=%w{up run}
-  OTHER_COMMANDS=%w{update proxy}
+  OTHER_COMMANDS=%w{update proxy cmd}
```
Comment (wgrrrr) [inline bin/dev]: Do you need to add `webinject` here?

File: bin/dev [hunk 3]
```diff
@@ -72,6 +74,22 @@ HEREDOC
...
+    container_id = `docker container ls -q -f name=ermanagement_web_`.strip
+    if container_id == ""
+      puts "Container is not running"
+    else
+      system("docker exec -i -t #{container_id} #{args.join(" ")}")
+    end
+  end
+
+  def cmd(args)
+    args = ["/bin/bash", "-l"] if args.empty?
```
Comment (wgrrrr) [inline bin/dev]: Just curious... what does `-l` do?

**Review by wgrrrr — CHANGES_REQUESTED** [review]
Looks good except for one question regarding `webinject`.

File: bin/dev (hunk 3 above)
Comment (philipwhittingham) [reply bin/dev]: i think it was something i had to add to get rvm to work, but i don't think these containers are using rvm so we can prolly remove it.

**Review by wgrrrr — APPROVED** [review]
👍

**Pull request merged**